templates/lb-fargate-service/manifest.yml

```diff
@@ -16,6 +16,8 @@ image:
 http:
   # Requests to this path will be forwarded to your service.
   path: '{{.Path}}'
+  # You can specify a custom health check path. The default is "/"
+  healthcheck: '{{.HealthCheckPath}}'
```

Comment:
What do you about having healthcheck commented out since it has a default of "/" and is not required?
```suggestion
#  # You can specify a custom health check path. The default is "/"
#  healthcheck: '{{.HealthCheckPath}}'
```

Reply:
done!